repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -407,6 +407,15 @@ def filter_out_out_pkgs(event_in_pkgs, event_out_pkgs):
     return {k: v for k, v in event_in_pkgs.items() if k not in event_out_pkgs}
 
 
+SKIPPED_PKGS_MSG = (
+    'packages will be skipped because they are available only in '
+    'RHEL 8 repositories that are intentionally excluded from the '
+    'list of repositories used during the upgrade. '
+    'See the report message titled "Excluded RHEL 8 repositories" '
+    'for details.\n The list of these packages (repositories: {}):'
```

Comment:
> The list of these packages (repositories: codeready-builder-for-rhel-8-x86_64-rpms):
> - python3-pyxattr

This is just a bit confusing. I'd change it to:
> The list of these packages together with the excluded repository they are available in:
> - python3-pyxattr: codeready-builder-for-rhel-8-x86_64-rpms

Reply:
This is nothing important/blocker. I suppose the majority would get the point.

Reply:
Created a ticket for this #552